examples/benchmarks/matmul.py

```diff
@@ -3,17 +3,16 @@
 
 """Model benchmark example for matmul with pytorch.
 
-Command to run:
+Commands to run:
```

Comment:
why "Commands"?

Reply:
Just to keep the same with other examples, like bert-large.